src/algebra/polynomial/big_operators.lean

```diff
@@ -85,7 +84,8 @@ section comm_ring
 variables [comm_ring R]
 
 open monic
---sort of a special case of Vieta?
+/-- Eventually this can be generalized with Vieta's formulas
+plus the connection between roots and factorization. -/
```

Comment:
A doc string should describe what the lemma proves. Implementation notes / TODOs should be ordinary block comments.

Reply:
like this?
```suggestion
-- Eventually this can be generalized with Vieta's formulas
-- plus the connection between roots and factorization.
```

Reply:
Yep! 👍